### DISCUSSION TITLE
working directory

### DISCUSSION THREAD

**Discussion by phntsm** [LeonardSSH/vscord#245] [Ideas]
Wouldn't mind the option of adding the current parent directory to whatever file you're working on.
`..{directory}/{file_name}{file_extension}:{current_line}:{current_column}`
Cheers! 🖤

**Comment by Khaomi** [comment]
Hey there! It already exists! `{directory_name}`

**Reply by phntsm** [reply]
Excellent! Hahaha, it's in the config list and I totally overlooked it - twice even. 😅
Loving it so far, thank you!
